### ISSUE TITLE
train loss

### ISSUE THREAD

**Issue by Zhang-jiemmd** [xinge008/Cylinder3D#78]
Hi,thank you for the great work!  
1.The loss function in this work is the same as the loss function in polarseg, right?
2.I used the same data (my own data set) to train polarseg and cylinder3d（grid size is [360,360,32]）, and found that the training loss of cylinder3d is relatively small. For example:
    epoch1 iter 2184   train loss(cylinder3d): 0.170     train loss(polarseg): 0.594   
                                    val loss(cylinder3d): 0.587        val loss(polarseg): 0.244   
    epoch15 iter 760   train loss(cylinder3d): 0.073     train loss(polarseg): 0.541
                                    val loss(cylinder3d): 0.248        val loss(polarseg): 0.291 
3.As mentioned in the second question, when I training cylinder3d, the training loss quickly drops to a value less than 0.1, but the validation set loss will be much larger than the training loss, and when I train for 20 epochs, the best car iou is only 86.6% (epoch11). When training on polarseg under the same settings, the best vehicle value is 96.42% (epoch14) .
4.Experimental setup(cylinder3d and polarseg):
   num class: 3 (not including the ignored 0 class)
   Training set: 19632 frames 
   val set: 3000 frames

**Comment by xinge008** [comment]
1. Yes
2. In the SemanticKITTI setting, the iou of car is about 97.2%; You can infer the pre-trained model with semanticKITTI validation set to check it. In your own setting, I have no idea why its car performance is only 86.6%.

BTW, these tailed categories in the validation set of SemanticKITTI maybe jittering, such as motorcycle, but not car.

**Comment by Runqing-forMost** [comment]
Hi, it is an excellent work!
In the released code, the training loss does not involve the point-wise CE loss, right?
